projects/vdk-core/.gitlab-ci.yml

```diff
@@ -129,6 +129,26 @@ vdk-core-release-candidate-base-image:
     changes: *vdk_core_locations
 
 
+vdk-core-release-acceptance-test:
+  stage: pre_release_test
+  before_script:
+    - cd projects/vdk-core
+  script:
+    - pip install vdk-heartbeat --extra-index-url $PIP_EXTRA_INDEX_URL
+    - export VDKCLI_OAUTH2_REFRESH_TOKEN=$VDK_API_TOKEN
+    - export VDK_HEARTBEAT_DEPLOY_JOB_VDK_VERSION=$(cat version.txt | grep -o '[0-9]\.[0-9]').${CI_PIPELINE_ID}.dev${CI_PIPELINE_IID}
```

Comment:
Out of curiosity, is there a specific reason why we use `CI_PIPELINE_IID` to specify the dev suffix? After looking at the linked CI/CD pipelines, it seems like this number can get pretty high. :)

Reply:
I took the job name directly from SC analogue CI/CD job. After digging a little on the subject, it seems to me that CI_PIPELINE_IID is supposed to be significantly smaller than CI_PIPELINE_ID and is actually suitable for versioning - https://gitlab.com/gitlab-org/gitlab-foss/-/issues/3691.
On the other hand, CI_PIPELINE_ID is completely unique and I don't see the need of appending another number after "dev". But I decided to keep the code consistent with SC. Maybe I should drop the second number, @doks5 @tozka, what do you think?

Reply:
It's fine to use it, I was just curious about the choice, as the number was four-digit. Nevertheless, it is generally expected that there is a number after "dev", as this is something package managers take into account.

Reply:
How can I add another auto-incrementing value per pipeline, do have suggestions?

Reply:
You can keep this one, it's not an issue.

Reply:
Thanks, Andy!